src/components/NavBtns/NavBtns.tsx

```diff
@@ -0,0 +1,88 @@
+import { useCallback } from "react";
+import styles from "./NavBtns.module.scss";
+
+export function NavBtns({
+  queryResults,
+  resultPageNumber,
+  setResultPageNumber,
+}: {
+  queryResults: any[];
+  resultPageNumber: number;
+  setResultPageNumber: any;
+}) {
+  let resultsPerPage = 10;
+  const totalResults = queryResults.length;
+  const resultsBehind = resultPageNumber * resultsPerPage;
+  const resultsCurrentAhead = totalResults - resultsBehind;
+
+  if (totalResults - resultsBehind < resultsPerPage) {
+    resultsPerPage = totalResults - resultsBehind;
+  }
+
+  const handleNavBtnClick = useCallback(
+    (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+      const numberOfSteps = Math.floor(totalResults / resultsPerPage);
+      const target = event.currentTarget;
+      const changeType = target.getAttribute("data-change-type");
```

Comment:
This is not ideal. It would be better for the `handleNavBtnClick` callback to accept an argument `changeType: "next" | "prewious"`

Reply:
ok done. pls see my last commit.